packages/client/src/v2-events/cache.ts

```diff
@@ -9,39 +9,35 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 
-import { FullDocumentURL, joinValues } from '@opencrvs/commons/client'
+import {
+  FullDocumentPath,
+  FullDocumentURL,
+  joinValues
+} from '@opencrvs/commons/client'
 
 /* Must match the one defined src-sw.ts */
 export const CACHE_NAME = 'workbox-runtime'
 
+export function getFullDocumentPath(filename: string): FullDocumentPath {
+  return ('/' +
```

Comment:
will the full document path always be an absolute path? (due to the leading slash). I'm asking this because when building the full url, if the base url has any paths attached, it would get replaced